src/erlcloud_ses.erl

```diff
@@ -1242,38 +1242,10 @@ decode_error(Doc) ->
 %%%------------------------------------------------------------------------------
 
 ses_request(Config, Action, Params) ->
-    case erlcloud_aws:update_config(Config) of
```

Comment:
Please don't delete this.

Reply:
You mean, keep call to `update_config`? It is called in `aws_request4`

https://github.com/erlcloud/erlcloud/blob/0f0540041ad6b55700715d1d3629853bf3158791/src/erlcloud_aws.erl#L184


And this is similar to what SQS code does:
https://github.com/erlcloud/erlcloud/blob/0f0540041ad6b55700715d1d3629853bf3158791/src/erlcloud_sqs.erl#L677-L678

Reply:
Oh, nice! I didn't realize that. Thanks